network/src/components/AppWrapper.js

```diff
@@ -587,6 +587,30 @@ export const AppWrapper = ({children}) => {
             onContextMenu={(e) => {
               e.preventDefault(); // prevent the default behaviour when right clicked
             }}>
+            <div className="button-container">
+              <button class='addPerson' onClick={() => {
+                  setPersonDialog(true);      // Open the "Add Person" dialog
+                  resetCanvasContextMenu();   // Reset the canvas context menu
+                  }} 
+                  style={{ padding: '10px 20px', 
+                    margin: '5px', 
+                    backgroundColor: 'green', 
+                    color: 'white'
+                  }}>
+                Add Person
+              </button>
+              <button class='addOrg' onClick={() => {
+                  setOrgDialog(true);      // Open the "Add Person" dialog
+                  resetCanvasContextMenu();   // Reset the canvas context menu
+                  }} 
+                  style={{ padding: '10px 20px', 
+                    margin: '5px', 
+                    backgroundColor: 'blue', 
+                    color: 'white' 
+                  }}>
+                Add Organization
+              </button>
+            </div>
```

Comment:
The functionality seems to be working correct here. However, the style looks a little odd with the placements of the buttons. I am not sure if this is how it is intended to look or perhaps it is different on my browser. 

<img width="1440" alt="image" src="https://github.com/user-attachments/assets/a8a99030-8477-4b2b-a5e5-4b6bfa0bdcb7">

Reply:
This is how it is on my side. I added in the description for eventual CSS changes.

Reply:
@simon-dunk Can we get this approved to just create a separate styling task?

Reply:
Task has been added to Asana